Userland/Libraries/LibC/wchar.cpp

```diff
@@ -687,13 +690,38 @@ size_t wcsspn(wchar_t const* wcs, wchar_t const* accept)
 }
 
 // https://pubs.opengroup.org/onlinepubs/9699919799/functions/wcsftime.html
-size_t wcsftime(wchar_t* __restrict wcs, size_t maxsize, wchar_t const* __restrict format, const struct tm* __restrict timeptr)
+size_t wcsftime(wchar_t* destination, size_t maxsize, wchar_t const* format, const struct tm* tm)
 {
-    (void)wcs;
-    (void)maxsize;
-    (void)format;
-    (void)timeptr;
-    dbgln("FIXME: Implement wcsftime()");
-    TODO();
+    // FIXME: Add actual wide char support for this.
+    char* ascii_format = static_cast<char*>(malloc(wcslen(format) + 1));
+    char* ascii_destination = static_cast<char*>(malloc(maxsize));
+
+    VERIFY(ascii_format && ascii_destination);
+
+    // These are copied by value because we will change the pointers without rolling them back.
+    AK::ScopeGuard free_ascii = [ascii_format, ascii_destination] {
```

Comment:
`AK::`?

Reply:
I accidentally left that in there when scratching my head over why `ScopeGuard` wasn't found (I forgot to include the header).

It's fixed now.